template/frameworks/ava/ava.config.cjs

```diff
@@ -1,5 +1,6 @@
 module.exports = () => {
   return {
+    files: ['./test/**/*.spec.js'],
```

Comment:
This will mean that for e2e as well as unit tests, users would have to have `.spec.js` in their file name

Reply:
Actually I think every real project will have fixed conveiton for tests naming, like `*.spec.js` or `*.test.js`, but your concern also makes sense.

I've moved `setup` to helpers which will be ingore by ava.